### PR TITLE
Releases and support doc

### LINKED ISSUE #20944: Overview doc for releases and patches for Fundamentals TOC
There is a [support policy page ](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) on the dotnet site, but it is more a reference resource than an introduction to or explanation of major/minor/patch releases and SDK feature bands. The new article can be based on this blog post: https://devblogs.microsoft.com/dotnet/net-core-releases-and-support/

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #20944 

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/core/releases-and-support?branch=pr-en-us-20978)

### CODE REVIEW

**Review by jamshedd — APPROVED** [review]

File: docs/fundamentals/toc.yml [hunk 1]
```diff
@@ -57,6 +57,8 @@ items:
     href: ../standard/class-libraries.md
   - name: .NET Standard overview
     href: ../standard/net-standard.md
+  - name: .NET releases, patches, and support
```
Comment (gewarren) [inline docs/fundamentals/toc.yml:60]:
```suggestion
  - name: Releases, patches, and support
```

File: docs/core/releases-and-support.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+title: .NET releases, patches, and support
+description: Learn about releases, patches, and support for .NET 5 (including .NET Core) and later versions.
+ms.date: 10/07/2020
+ms.topic: overview
+ms.author: tdykstra
+author: tdykstra
+---
+# Releases and support for .NET Core and .NET 5
+
+Microsoft ships major releases, minor releases, and servicing updates (patches) for .NET Core, .NET 5, and later versions. This article explains release types, servicing updates, SDK feature bands, support periods, and support options.
```
Comment (gewarren) [inline docs/core/releases-and-support.md:11]:
```suggestion
Microsoft ships major releases, minor releases, and servicing updates (patches) for .NET Core and .NET 5.0 and later versions. This article explains release types, servicing updates, SDK feature bands, support periods, and support options.
```

File: docs/core/releases-and-support.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+---
+title: .NET releases, patches, and support
```
Comment (gewarren) [inline docs/core/releases-and-support.md:2]: Do we want to put this on the landing page?

File: docs/core/releases-and-support.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+### Servicing updates
+
+Servicing updates (patches) ship almost every month, and these updates carry both security and non-security bug fixes. For example, .NET Core 3.1.8 is the eighth update for .NET Core 3.1. When these updates include security fixes, they're released on "patch Tuesday", which is always the second Tuesday of the month. Servicing updates are expected to maintain compatibility. Starting with .NET Core 3.1, servicing updates are upgrades that remove the preceding update. For example, the latest servicing update for 3.1 removes the previous 3.1 update upon successful installation.
+
+### Feature bands (SDK only)
+
+Versioning for the .NET SDK works slightly differently from the .NET runtime. .NET SDK updates sometimes include new features or new versions of components like MSBuild and NuGet to align with new Visual Studio releases. These new features or components may be incompatible with the versions that shipped in previous SDK updates for the same major or minor version.
```
Comment (gewarren) [inline docs/core/releases-and-support.md:37]:
```suggestion
Versioning for the .NET SDK works slightly differently from the .NET runtime. To align with new Visual Studio releases, .NET SDK updates sometimes include new features or new versions of components like MSBuild and NuGet. These new features or components may be incompatible with the versions that shipped in previous SDK updates for the same major or minor version.
```

File: docs/core/releases-and-support.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+  These versions are supported for a minimum of 3 years, or 1 year after the next LTS release ships if that date is later.
+
+  Example:
+
+  * .NET Core 2.1 was released in May 2018 and was deemed an LTS release in August  2018.
+  * .NET Core 3.1 was the next LTS release and was released in December  2019.
+  * Because August 2021 (3 years) is later than December  2020 (one year after the 3.1 release), .NET Core 2.1 is supported through August 2021.
+
+Releases alternate between LTS and Current, so it's possible for an earlier release to be supported longer than a later release. For example, .NET Core 2.1 is an LTS release with support through August  2021. The 3.0 release shipped more than a year later but went out of support earlier, in December  2019.
```
Comment (gewarren) [inline docs/core/releases-and-support.md:74]:
```suggestion
Releases alternate between LTS and current, so it's possible for an earlier release to be supported longer than a later release. For example, .NET Core 2.1 is an LTS release with support through August  2021. The 3.0 release shipped more than a year later but went out of support earlier, in December  2019.
```

Comment (gewarren) [reply docs/core/releases-and-support.md:74]: I see you capitalize it elsewhere so maybe leave as-is.

File: docs/core/releases-and-support.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+For information about the latest servicing updates for each major and minor version, see the [.NET downloads page](https://dotnet.microsoft.com/download/dotnet-core).
+
+## End of support
+
+End of support refers to the date after which Microsoft no longer provides fixes, updates, or technical assistance for a product version. Before this date, make sure you have moved to using a supported version. Versions that are out of support no longer receive security updates that protect your applications and data.
+
+## Supported operating systems
+
+.NET Core and .NET 5 can be run on a range of operating systems. Each of these operating systems has a lifecycle defined by its sponsor organization (for example, Microsoft, Red Hat, or Apple). We take these lifecycle schedules into account when adding and removing support for operating system versions.
```
Comment (gewarren) [inline docs/core/releases-and-support.md:98]:
```suggestion
.NET Core and .NET 5+ can be run on a range of operating systems. Each of these operating systems has a lifecycle defined by its sponsor organization (for example, Microsoft, Red Hat, or Apple). We take these lifecycle schedules into account when adding and removing support for operating system versions.
```

File: docs/core/releases-and-support.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+When an operating system version goes out of support, we stop testing that version and providing support for that version. Users need to move forward to a supported operating system version to get support.
+
+For more information, see the [.NET OS Lifecycle Policy](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md).
+
+## Getting Support
```
Comment (gewarren) [inline docs/core/releases-and-support.md:104]:
```suggestion
## Get support
```

File: docs/core/releases-and-support.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+You have a choice between Microsoft assisted support and Community support.
+
+### Microsoft Support
```
Comment (gewarren) [inline docs/core/releases-and-support.md:108]:
```suggestion
### Microsoft support
```

File: docs/core/releases-and-support.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+For assisted support with .NET Core and .NET 5, [contact a Microsoft Support Professional](https://support.microsoft.com/supportforbusiness/productselection/?sapid=4fd4947b-15ea-ce01-080f-97f2ca3c76e8).
```
Comment (gewarren) [inline docs/core/releases-and-support.md:110]:
```suggestion
For assisted support with .NET Core and .NET 5+, [contact a Microsoft Support Professional](https://support.microsoft.com/supportforbusiness/productselection/?sapid=4fd4947b-15ea-ce01-080f-97f2ca3c76e8).
```

File: docs/core/releases-and-support.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+
+You need to be on a supported servicing level (the latest available servicing update) to be eligible for support. If a system is running 3.1 and the 3.1.8 servicing update has been released, then 3.1.8 needs to be installed as a first step.
+
+### Community Support
```
Comment (gewarren) [inline docs/core/releases-and-support.md:114]:
```suggestion
### Community support
```

**Review by gewarren — APPROVED** [review]
Left some suggestions for you to consider.

**Comment by tdykstra** [comment]
@gewarren Thanks for the review!

**Pull request merged**